lib/src/client/client.dart

```diff
@@ -33,4 +33,8 @@ class Client {
       {CallOptions options}) {
     return _channel.createCall(method, requests, _options.mergedWith(options));
   }
+
+  Future<void> close() async {
```

Comment:
Should this be called `shutdown()`?
Should we also have a `terminate()`?
Also this needs a doc-comment.

I guess one can construct several clients from the same channel - so when closing the client it is surprising that it closes them all. No?

Would it be better to expose the channel itself?

Reply:
Several clients over the same channel isn't a usecase I'd imagined. I'll reconfigure `appengine` to hold onto the client channel and shut it down as appropriate.